ci-operator/step-registry/openshift-extended/test/disasterrecovery/openshift-extended-test-disasterrecovery-ref.yaml

```diff
@@ -0,0 +1,61 @@
+ref:
+  as: openshift-extended-test-disasterrecovery
+  from: tests-private
+  grace_period: 10m
+  commands: openshift-extended-test-disasterrecovery-commands.sh
+  cli: latest
+  timeout: 22h0m0s
+  env:
+  - name: TEST_SCENARIOS_DISASTERRECOVERY
+    default: DR_Testing
+    documentation: |-
+      Select the test case. It's a RE pattern and is passed to the 'grep -E'
+  - name: TEST_ADDITIONAL_DISASTERRECOVERY
+    default: ""
+    documentation: |
+      It is additional scenario to combine the TEST_SCENARIOS_DISASTERRECOVERY to make final scenarios.
+  - name: TEST_FILTERS_DISASTERRECOVERY
+    default: "~ChkUpgrade&;NonPreRelease&;~DisconnectedOnly&;~HyperShiftMGMT&;~MicroShiftOnly&"
+    documentation: |
+      It is filters to select case based on some case attribution which is in case title.
+  - name: FILTERS_ADDITIONAL_DISASTERRECOVERY
+    default: "~CPaasrunOnly&"
+    documentation: |
+      It is additional filters to combine the TEST_FILTERS_DISASTERRECOVERY to make final filters.
+  - name: MODULE_FILTERS_DISASTERRECOVERY
+    default: ""
+    documentation: |
+      After test filsters are executed, it is filters to select case for certain moudle.
+      the formart: <moudle_name1>:<filters1>#<moudle_name2>:<fitlers2>#<moudle_nameN>:<fitlersN>
+      the example: OLM:V1&;~Catalog&#Operator_SDK:~opm&;Cve&
+  - name: TEST_IMPORTANCE
+    default: ""
+    documentation: it means case IMPORTANCE.
+  - name: TEST_TIMEOUT_DISASTERRECOVERY
+    default: "75"
+    documentation: it means the duration of one case execution.
+  - name: TEST_PARALLEL
+    default: "6"
+    documentation: it means the number of case execution in parallel.
+  - name: TEST_ROSA_LOGIN_ENV
+    default: "staging"
+    documentation: The environment for rosa login. The supported values are [production, staging].
+  - name: FORCE_SUCCESS_EXIT
+    default: "yes"
+    documentation: |-
+      Setting this to 'yes' will allow some statements in the scripts not to let the step fail, and further steps can
+      run. Setting it to 'no' will disable this behavior, letting any of those optional to succeed statements 
+      to stop the job and trigger the deprovisioning steps. Default: yes.
+      See https://docs.ci.openshift.org/docs/architecture/step-registry/#workflow
+  credentials:
```

Comment:
@geliu2016 
```yaml
  credentials:
  - namespace: test-credentials
    name: ci-route-53
    mount_path: /var/run/vault/aws
```
should not needed.

it is used for special case which is not DR case.

Reply:
Updated.